launch_config.tf

```diff
@@ -0,0 +1,135 @@
+data "aws_ami" "ecs" {
+  most_recent = true
+  name_regex  = "^amzn2-ami-ecs-hvm-.*-x86_64-ebs"
```

Comment:
This should probably be a var. There are other ECS AMIs one could use if one was into such things.

Reply:
Come to think about it, I think you should leave this as is but then let the user specify an AMI ID and ignore this.